### PR TITLE
CPNHUB-94 FIX: subscribers required true.

### PR DESCRIPTION
**Pull request by dev-tsoni**

### CODE REVIEW

File: newsroom/topics/topics.py [hunk 1]
```diff
@@ -19,7 +19,8 @@ class TopicsResource(newsroom.Resource):
         'is_global': {'type': 'boolean', 'default': False},
         'subscribers': {
             'type': 'list',
-            'schema': newsroom.Resource.rel('users', required=True)
+            'schema': newsroom.Resource.rel('users'),
+            'required': True
```
Comment (petrjasek) [inline newsroom/topics/topics.py:23]: why are subscribers required?

File: assets/components/AuditInformation.jsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ class AuditInformation extends React.Component {
         const users = get(this.props, 'users.length') ? this.props.users : this.props.editUsers;
 
         if (field === '_created' && item._created) {
-            const creator = getItemFromArray(item.original_creator,
+            const creator = getItemFromArray(item.users,
```
Comment (petrjasek) [inline assets/components/AuditInformation.jsx:31]: why this change?

File: newsroom/topics/topics.py (hunk 1 above)
Comment (dev-tsoni) [reply newsroom/topics/topics.py]: There was an issue while updating the topics. Where subscribers was required. Fixed it and updated the PR.

File: assets/components/AuditInformation.jsx (hunk 2 above)
Comment (dev-tsoni) [reply assets/components/AuditInformation.jsx]: This was done earlier when `original_creator` was not updated when we created topics via API. Since now it is updated as per the user, this is not required.

**Review by petrjasek — APPROVED** [review]
